gnovm/cmd/gno/test.go

```diff
@@ -246,6 +255,10 @@ func execTest(cfg *testCfg, args []string, io commands.IO) error {
 			io.ErrPrintfln("FAIL    %s \t%s", pkg.Dir, dstr)
 			io.ErrPrintfln("FAIL")
 			testErrCount++
+			if cfg.failfast {
+				io.ErrPrintfln("FAIL")
```

Comment:
there is already a printed `FAIL` two lines ago.

Reply:
Hey thanks a lot for your review :)
I think there was some mix with the files diffs but I'm doing the same behavior that on lines
https://github.com/gnolang/gno/pull/3866/files#diff-2e39c5441249018f694183a5710aa7adcad09ec17e870507d20a42184b07e1a7R278-R281

Reply:
can you merge the code with the one at the end of the function, maybe with a closure?

Reply:
Did you meant something like this ?
cb4ee65fa8a42d5fc01fe79be1660b1367c0d937